src/main/java/busTickets/BusTicketValidation.java

```diff
@@ -0,0 +1,38 @@
+package busTickets;
+
+import java.time.LocalDate;
+
+public class BusTicketValidation {
+
+    public static boolean validateBusTicketStartDate(LocalDate busTicketStartDate, BusTicketType busTicketType) {
+        if (busTicketStartDate == null && (busTicketType == BusTicketType.DAY || busTicketType == BusTicketType.WEEK || busTicketType == BusTicketType.MONTH || busTicketType == BusTicketType.YEAR)) {
+            BusTicket.startDateViolation++;
```

Comment:
don't use static fields for saving state, create separate variables and operate with them

Reply:
Resolved